src/components/page-configuration/use-regenerate-deploy-key-and-secret.js

```diff
@@ -1,9 +1,11 @@
+import { nanoid } from 'nanoid';
+
 import { usePostJson } from '../../effects';
-var phraseit = require('phraseit');
 
-const useRegenerateDeployKeyAndSecret = (appName) => {
+export const useRegenerateDeployKeyAndSecret = (appName) => {
```

Comment:
This function is only used by the configure-application-github component. Perhaps we should move it there.

Reply:
moved into appropriate folder